testing/ssh-honeypot/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="ssh-honeypot"
+pkgver="0.1.0"
+_commit=b7baa9a6963e723ee0779a6cef0447739870ace3
+pkgrel=0
+pkgdesc="Fake sshd that logs ip addresses, usernames, and passwords."
+url="https://github.com/droberson/ssh-honeypot"
+arch="all"
+license="MIT"
+depends="openssh-keygen"
+makedepends="clang json-c0.12-dev libssh-dev"
```

Comment:
Why the dependency on clang?

Reply:
replaced with gcc

Reply:
That should be in there by default, so no need to depend on that.

Reply:
Ah you did that, my bad